### PR TITLE
[release-4.15] OCPBUGS-33283: Adjust NAD name to "network-xxx-xxx" when creating

### PR DESCRIPTION
**Pull request by avivtur**
Manual backport to https://github.com/openshift/console/pull/13568

### CODE REVIEW

**Comment by jhadvig** [comment]
```
[3/5] Fetching packages...
warning Pattern ["asn1js@latest"] is trying to unpack in the same destination "/usr/local/share/.cache/yarn/v6/npm-asn1js-2.0.26-0a6d435000f556a96c6012969d9704d981b71251-integrity/node_modules/asn1js" as pattern ["asn1js@^2.0.26"]. This could result in non-deterministic behavior, skipping.
error An unexpected error occurred: "https://registry.yarnpkg.com/@babel/runtime-corejs2/-/runtime-corejs2-7.14.8.tgz: Request failed \"502 Bad Gateway\"".
```

**Review by jhadvig — APPROVED** [review]
/approve
/label backport-risk-assessed

File: frontend/packages/network-attachment-definition-plugin/src/components/network-attachment-definitions/NetworkAttachmentDefinitionsForm.tsx [hunk 1]
```diff
@@ -101,8 +104,8 @@ const getResourceName = (networkType, typeParamsData): string => {
     : `openshift.io/${_.get(typeParamsData, 'resourceName.value', '')}`;
 };
 
-const generateNADName = (name: string): string => {
-  return `${name}-${uniqueNamesGenerator({
+const generateNADName = (): string => {
+  return `network-${uniqueNamesGenerator({
```
Comment (lkladnit) [inline frontend/packages/network-attachment-definition-plugin/src/components/network-attachment-definitions/NetworkAttachmentDefinitionsForm.tsx:108]: Can it be `nad-${uniqueNamesGenerator... ?
Or `netattdef-${uniqueNamesGenerator... ?
Common word "network" with randomized suffix is difficult to test for

File: frontend/packages/network-attachment-definition-plugin/locales/en/kubevirt-plugin.json [hunk 2]
```diff
@@ -14,12 +14,12 @@
   "Network Type": "Network Type",
   "Edit YAML": "Edit YAML",
   "Networks are not project-bound. Using the same name creates a shared NAD.": "Networks are not project-bound. Using the same name creates a shared NAD.",
-  "Create Network Attachment Definition": "Create Network Attachment Definition",
   "Name": "Name",
   "Description": "Description",
   "<0>OpenShift Virtualization Operator</0> or <3>SR-IOV Network Operator </3>needs to be installed on the cluster, in order to pick the Network Type.": "<0>OpenShift Virtualization Operator</0> or <3>SR-IOV Network Operator </3>needs to be installed on the cluster, in order to pick the Network Type.",
   "Create": "Create",
   "Cancel": "Cancel",
+  "Create Network Attachment Definition": "Create Network Attachment Definition",
```
Comment (lkladnit) [inline frontend/packages/network-attachment-definition-plugin/locales/en/kubevirt-plugin.json:22]: It's supposed to be "Create NetworkAttachmentDefinition", isn't it?

**Pull request merged**